UncertainSCI/distributions.py

```diff
@@ -174,15 +182,66 @@ def MC_samples(self, M=100):
 
         return self.transform_to_standard.mapinv(p)
 
+    # def means(self):
```

Comment:
Why are these commented out? Anyway, `NormalDistribution` does not have `mean` and `cov` methods, but the other distributions, `ExponentialDistribution` and `BetaDistribution` do. We'll need `mean` and `cov` methods for `NormalDistribution` for consistency.